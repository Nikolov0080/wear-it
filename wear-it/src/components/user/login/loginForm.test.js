import React from 'react';
import renderer from 'react-test-renderer';
import LoginForm from './loginForm';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Login', () => {
    const tree = renderer
        .create(<BrowserRouter><LoginForm /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});