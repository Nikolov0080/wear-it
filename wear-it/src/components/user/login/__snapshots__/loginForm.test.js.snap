// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Login 1`] = `
<div
  className="box"
>
  <div
    className="loginForm"
    onChange={[Function]}
  >
    <form
      onSubmit={[Function]}
    >
      <h2
        className="title"
      >
        LOGIN
      </h2>
      
      <div>
        <input
          autoComplete="off"
          className="input"
          name="username"
          onChange={[Function]}
          placeholder="Username"
          type="text"
        />
      </div>
      <div>
        <input
          autoComplete="off"
          className="input"
          name="password"
          onChange={[Function]}
          placeholder="Password"
          type="password"
        />
      </div>
      <div>
        <button
          className="btn"
          type="submit"
        >
          Submit
        </button>
      </div>
    </form>
  </div>
</div>
`;
